src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return "Color: " + color + ", Number: " + number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+}
```

Comment:
According to the checklist, classes, fields, constructors, methods must have access modifiers. In this case, consider adding access modifiers to the fields 'color' and 'number' to encapsulate the state of the Ball object properly.